source/blog/2015-03-01-cappuccino-news-2015-03-01.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: Cappuccino News 2015 March 1st
+author: Alexandre Wilhelm
+author_email: alexandre.wilhelmfr@gmail.com
+date: '2015-03-01'
+tags:
+- Cappuccino
+- Cucapp
+- Cucumber
+- Objective-J
+- update
+- OJTest
+---
+
+Here's a quick update on what's new with Cappuccino for the last month.
+
+### Cappuccino
+
+##### News
+
+- The graphical `CPDatePicker` now disables the dates outside the `minDate` and `maxDate`. ([daboe01](https://github.com/daboe01) - [PR](https://github.com/cappuccino/cappuccino/pull/2297))
+
+- `Cappuccino` now draws by default in high DPI in a `canvas2D`. You can disable this feature using the `CPView` class method `setAllowsHighDPIDrawing:`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2315))
+
+- The method `setNeedsLayout:` and `needsLayout` has been added to the class `CPView`. You can now easily control to lay out or not a view. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2316))
+
+- `Cappuccino` now supports the `KVC` operators `unionOfObjects`, `distinctUnionOfObjects`, `unionOfArrays`, `distinctUnionOfArrays` and `distinctUnionOfSets`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2313))
+
+- The method `viewAtColumn:row:makeIfNecessary` has been added to the class `CPTableView`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2315))
+
+##### Changes/Fixes
+
+- The method `setValue:forKey:` of the `CPObjectController` does not use the `theKeyPath` parameter as `keyPath` and `value` anymore. ([krodelin](https://github.com/krodelin) - [PR](https://github.com/cappuccino/cappuccino/pull/2301))
+
+- Changed the name of the method `CPDomDisplayServerSetStyleBackgroundSize` to `CPDOMDisplayServerSetStyleBackgroundSize`. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2296))
+
+- Removed unreachable code in `CPDOMDisplayServer.h`. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2300))
+
+- The `CPStepper` class method returns `id` instead of `CPStepper` type. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2294))
```

Comment:
The \`CPStepper\` class initializer methods return an object of type \`id\` instead of \`CPStepper\`.